README.md

```diff
@@ -70,7 +70,13 @@ The React components are built using the [TypeScript](https://www.typescriptlang
 
 ## 💻 Getting started
 
-As a prerequisite to run this app you will need to install [Node.js](https://nodejs.org/en/download)(version 16.20.2 is currently used in local development and in the pipeline) and [npm](https://docs.npmjs.com/cli/v7/commands/npm-install):
+As a prerequisite to run this app you will need to install [Node.js](https://nodejs.org/en/download)(version 20.17.0 is currently used in local development and in the pipeline) and [npm](https://docs.npmjs.com/cli/v10/commands/npm-install):
+
+If you have Node Version manager you can set node to the correct version using the nvm command.
+
+```
+nvm use
+```
 
 ### Running locally
 
```

Comment:
total noob dyslexic note, would be good to bring up the 127.0.0.1   localdev.hackney.gov.uk to this chain of commands with both win/mac cmd line to do so